src/System.Reflection.Metadata/src/System/Reflection/Metadata/PortablePdbReaderExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Reflection.Internal;
+using System.Reflection.PortablePdb;
+using System.Text;
+
+namespace System.Reflection.Metadata
+{
+    // EditorBrowsable(Never) so that we don't clutter completion list with this type because a user that only has System.Reflection.Metadata 
+    // imported and has type PE is likely looking to resolve PEReader from the System.Reflection.PortableExecutable and not looking to invoke
+    // these extensions as regular statics.
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
This comment says specific things about PortableExecutableReader PE (copied from PEReaderExtensions) no doubt. Should we keep EditorBrowsable(never) here. I'm OK with that for consistency, but I'd clarify the comment.